<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
font: 87 12pt &quot;Arial Black&quot;;
color: rgb(20, 200,220);
}

QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>353</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Coordenadas Esféricas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_cerrar">
            <property name="text">
             <string>Cerrar</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>353</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="styleSheet">
             <string notr="true">QLabel{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}

QDoubleSpinBox{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,1,1,1">
             <item>
              <widget class="QTextEdit" name="textEdit">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000; background-color:#ffffff;&quot;&gt;Las coordenadas esféricas constituyen otra generalización de las coordenadas polares del plano, a base de girarlas alrededor de un eje. Su definición es la siguiente:&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot; align=&quot;justify&quot; style=&quot; margin-top:10px; margin-bottom:0px; margin-left:20px; margin-right:10px; -qt-block-indent:0; text-indent:0px;&quot;&gt;La coordenada &lt;span style=&quot; font-style:italic;&quot;&gt;radial&lt;/span&gt; &lt;span style=&quot; font-style:italic;&quot;&gt; &amp;quot;&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;r&amp;quot;&lt;/span&gt;: distancia al origen&lt;/li&gt;
&lt;li style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot; align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:20px; margin-right:10px; -qt-block-indent:0; text-indent:0px;&quot;&gt;La coordenada &lt;span style=&quot; font-style:italic;&quot;&gt;polar&lt;/span&gt;  &amp;quot;&lt;span style=&quot; font-family:'Google Sans,arial,sans-serif'; font-size:20px; color:#202124; background-color:#ffffff;&quot;&gt;θ&amp;quot;&lt;/span&gt;: ángulo que el vector de posición forma con el eje Z.&lt;/li&gt;
&lt;li style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot; align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:10px; margin-left:20px; margin-right:10px; -qt-block-indent:0; text-indent:0px;&quot;&gt;La coordenada &lt;span style=&quot; font-style:italic;&quot;&gt;acimutal&lt;/span&gt;  &amp;quot;&lt;span style=&quot; font-family:'Google Sans,arial,sans-serif'; font-size:16px; color:#4d5156; background-color:#ffffff;&quot;&gt;φ&amp;quot;&lt;/span&gt;: ángulo que la proyección sobre el plano XY forma con el eje X.&lt;/li&gt;&lt;/ul&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:18px; margin-bottom:18px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000; background-color:#ffffff;&quot;&gt;El uso más evidente de las coordenadas esféricas lo constituye la geografía. Para identificar un punto de la superficie terrestre indicamos su latitud y su longitud.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:18px; margin-bottom:18px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000; background-color:#ffffff;&quot;&gt;La&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; font-style:italic; color:#000000;&quot;&gt;latitud&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt; es la altura respecto al ecuador. Este ángulo es el complementario de la coordenada polar &lt;/span&gt;&lt;span style=&quot; font-family:'Google Sans,arial,sans-serif'; font-size:11pt; color:#202124; background-color:#ffffff;&quot;&gt;θ&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt; (por lo cual a ésta se la llama también &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; font-style:italic; color:#000000;&quot;&gt;colatitud&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt;). La latitud, en lugar de variar de 0 (en el Polo Norte) a &lt;/span&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:11pt; color:#5f6368; background-color:#ffffff;&quot;&gt;π&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt; (en el Polo Sur) lo hace desde 90° a -90°.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:18px; margin-bottom:18px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000; background-color:#ffffff;&quot;&gt;La&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; font-style:italic; color:#000000;&quot;&gt;longitud&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt; es la distancia angular respecto a un meridiano fijo (el de Greenwich). Equivale a la coordenada acimutal &lt;/span&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:11pt; color:#4d5156; background-color:#ffffff;&quot;&gt;φ&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:18px; margin-bottom:18px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000; background-color:#ffffff;&quot;&gt;La coordenada radial corresponde a la distancia al centro de la Tierra. La altitud&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt; z de un punto de la superficie equivale al valor de r = z + R&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000; vertical-align:sub;&quot;&gt;T&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt; con R&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000; vertical-align:sub;&quot;&gt;T&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto Slab,Cambria,Lucida Grande,Verdana,Arial,sans-serif'; font-size:11pt; color:#000000;&quot;&gt; el radio de la Tierra (suponiendo ésta una esfera, lo que es solo una aproximación).&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Inserte las coordenadas:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="1">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Y:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Z:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>X:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QDoubleSpinBox" name="sBox_X">
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="minimum">
                  <double>-100000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="sBox_Y">
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="minimum">
                  <double>-100000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QDoubleSpinBox" name="sBox_Z">
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="minimum">
                  <double>-100000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="button_simular">
               <property name="text">
                <string>Simular</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="graf_esfe"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
